easybuild/tools/filetools.py

```diff
@@ -177,7 +200,7 @@ def remove_file(path):
         if os.path.exists(path):
             os.remove(path)
     except OSError, err:
-          _log.error("Failed to remove %s: %s", path, err)
+            _log.error("Failed to remove%s: %s" % (path, err))
```

Comment:
yes, EasyBuildLog breaks the way logger.error should work, so I had to resort to this workaround, will have to be fixed in EasyBuildLog in a separate issue.